manifests/08-clusteroperator.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
```

Comment:
Fyi, the correct name is `console`, though the `console.config.openshift.io` resource is `cluster` as is `console.operator.openshift.io`.